common/src/main/java/io/homeassistant/companion/android/database/widget/graph/GraphWidgetEntity.kt

```diff
@@ -0,0 +1,31 @@
+package io.homeassistant.companion.android.database.widget.graph
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import io.homeassistant.companion.android.database.widget.ThemeableWidgetEntity
+import io.homeassistant.companion.android.database.widget.WidgetBackgroundType
+import io.homeassistant.companion.android.database.widget.WidgetEntity
+import io.homeassistant.companion.android.database.widget.WidgetTapAction
+
+@Entity(tableName = "graph_widget")
+data class GraphWidgetEntity(
+    @PrimaryKey
+    override val id: Int,
+    @ColumnInfo(name = "server_id", defaultValue = "0")
+    override val serverId: Int,
+    @ColumnInfo(name = "entity_id")
+    val entityId: String,
+    @ColumnInfo(name = "label")
+    val label: String?,
+    @ColumnInfo(name = "graph_time_range")
+    val timeRange: Int = 24,
+    @ColumnInfo(name = "tap_action", defaultValue = "REFRESH")
+    val tapAction: WidgetTapAction,
+    @ColumnInfo(name = "last_update")
+    val lastUpdate: Long,
+    @ColumnInfo(name = "background_type", defaultValue = "DAYNIGHT")
```

Comment:
This should not include a `defaultValue`, as it is always set by the widget. It exists on other widget data classes for data migration purposes.

Reply:
I had to override it , so I want it to keep it consistant

Reply:
It is an override so I'd say that explicitly means you're allowed to change it 😅 My suggestion was only for the annotation (which is used by Room to set the value on the table, which isn't needed here!), not for the Kotlin variable.